docker-compose.yml

```diff
@@ -21,18 +21,21 @@ services:
       context: .
       dockerfile: Dockerfile.webmon
       # build using the host's network, required in certain environments when accessing the debian repositories
+      network: host
```

Comment:
Why?

Reply:
I build the apps in `ndav4.sns.gov` and docker cannot "reach out" to remote package repositories when creating the images. This is the only time when the host network is used, once the images are created the host network is not used